tests/integration-tests/tests/efa/test_efa.py

```diff
@@ -197,8 +202,19 @@ def _test_osu_benchmarks_collective(
 
 
 def _test_osu_benchmarks_multiple_bandwidth(
-    remote_command_executor, scheduler_commands, test_datadir, slots_per_instance, partition=None
+    instance, remote_command_executor, scheduler_commands, test_datadir, slots_per_instance, partition=None
 ):
+    instance_bandwidth_dict = {
```

Comment:
Minor not blocking, can you add the trn1.32xlarge performance?
8 NIC of 100 Gbps?

Reply:
sure, done